auth0/src/main/java/com/auth0/android/authentication/AuthenticationAPIClient.kt

```diff
@@ -923,26 +923,25 @@ public class AuthenticationAPIClient @VisibleForTesting(otherwise = VisibleForTe
     }
 
     /**
-     * Creates a new request to fetch a session token in exchange for a refresh token.
+     * Creates a new request to fetch a web sso token in exchange for a refresh token.
```

Comment:
Should this be `Creates a new request to fetch a session transfer token in exchange for a refresh token.`?